djaopsp/urls/views/reports.py

```diff
@@ -27,6 +27,8 @@
          AssessPracticesXLSXView.as_view(), name='assess_download_segment'),
     path('assess/<slug:sample>/download/',
          AssessPracticesXLSXView.as_view(), name='assess_download'),
+    path('assess/<slug:sample>/downloadpptx/',
```

Comment:
I would prefer path `assess/<slug:sample>/download/pptx/` such that when reading through the HTTP request logs, it is simpler to filter "download" URLs.